wm/frame.go

```diff
@@ -356,6 +359,11 @@ func (f *frame) applyTheme(force bool) {
 	}
 }
 
+func (f *frame) setTitle() {
+	//Should we somehow just be changing the label rather than forcing a theme apply?
```

Comment:
This is not really a code document - more like a TODO or question, does it belong here?
If this is not the right way to do something should it be pushed?

p.s. Go likes a space after the slashes ;)

Reply:
Removed the comment - After looking at the code is proper because applyTheme() handles refresh rather completely redoing the theme.